standup_notes/resources/standup-notes.bash

```diff
@@ -2,36 +2,32 @@
 
 _standup_completions()
 {
-    local cur prev opts
+    local cur prev opts base
```

Comment:
where is `base` used? Also, should `prev2` and `days` be local as well?

Reply:
base is not being used, going to remove it. I have also added prev2 and days to be local